### DISCUSSION TITLE
停止使用 Gitee 的通知

### DISCUSSION THREAD

**Discussion by lisonge** [gkd-kit/gkd#263] [General]
gkd 已经关闭并注销所有在大陆的网页，未来将只使用 gkd.li 域下的域名，也不再考虑主动在大陆部署网页

但 gkd 所有配套工具均为开源，您可以自行 clone 自行任意部署

对于原来如 <https://gkd-kit.gitee.io/import/13263321> 显示 404 的问题，您只需要将 `gkd-kit.gitee.io` 替换为 `i.gkd.li` 即可

以下是一个简单的油猴脚本来帮助您自动重定向到 `i.gkd.li`

```js
// ==UserScript==
// @name        To-i.gkd.li
// @namespace   Violentmonkey Scripts
// @match       https://gkd-kit.gitee.io/import/*
// @grant       none
// @version     1.0
// @author      -
// @description 2023/11/9 23:13:16
// ==/UserScript==

const u = new URL(location.href);
u.host = 'i.gkd.li';
location.href = u.href;
```

**Comment by lisonge** [comment]
手动 at 一下所有订阅的贡献者

@Snoopy1866 @jiuqianyuan @walklty @aoguai @xiaowonet @CMBill @dlh996 @yanyu2023 @shanlanCoding @BreezyOrigin @Sec-ant @Otis0227 @longm21 @OldCoding @xe5700 @YaChengMu @shjdgwj @zhyx490991014 @Nebulafalls @lliioollcn @g1fts @zhangyusheng211 @xuanqb @MoeGrid @kyrinyoga @sherylynn @wjf0214 @Ran0929 @javabcde @ZhReimu @zhuzhuani @sishuikaki @maxchang3 @xhemj @ysjsgzq @Zonezzc @guanyintu @alzuobaba @marryton007 @ylxz95 @Schweik7 @w4616 @lsxuan12138 @small-ora @koo6666 @Long-Miao @zhcode-fun @someone120

如果这条消息打扰到您，麻烦请忽略

**Comment by boy86001** [comment]
大老万岁🤩
